spec/models/miq_expression_spec.rb

```diff
@@ -1631,4 +1631,146 @@
       end
     end
   end
+
+  describe "#sql_supports_atom?" do
+    context "expression key is 'CONTAINS'" do
+      context "operations with 'tag'" do
+        it "returns true if format of field is /\w+\.\w+/ - interpreted as tag of the main model" do
```

Comment:
* WDYT about something like `it "returns true for tags of the main model"` here? I think this is a great place to describe the high level behavior...letting implementation details in here can set themselves up to be future lies
* Did you consider using "tag" in place of "field" here? We are dealing with tags, and although we have used the two almost interchangeably in other places I think it would be good to start drawing a line between the two